app/views/code_reviews/index.html.erb

```diff
@@ -1,19 +1,17 @@
 <h1>
   Code Reviews
+  <% if can? :manage, CodeReview %>
+    <%= link_to '+ New Code Review', new_code_review_path, class: 'text-success new-link' %>
+  <% end %>
```

Comment:
Also, the same applies for new internships and companies. I'm trying to keep the design consistent, whatever it ends up being.